Cakes-Co/style.css

```diff
@@ -1 +1,257 @@
 /*  Add your styling here */
+
+*{
+    margin: 0;
+    box-sizing: border-box;
+    padding: 0;
+}
+
+body{
+
+    
+    font-family: "Poppins";
+}
+
+.header{
+background-image: url("https://images.pexels.com/photos/7966383/pexels-photo-7966383.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" );
+background-position: center;
+background-size: cover;
+background-repeat: no-repeat;
+padding: 150px;
+position:relative;
+z-index: 2;
```

Comment:
This `z-index: 2;` is finally unnecessary in this case. It does not affect anything. [Here is a good read](https://www.joshwcomeau.com/css/stacking-contexts/) if you want to dive in.